internal/backends/nodejs/grab.go

```diff
@@ -163,7 +164,7 @@ func guessBareImports() map[api.PkgName]bool {
 
 			isInternalMod := false
 			for _, internal := range internalModules {
-				if internal == mod {
+				if internal == mod || strings.HasPrefix(mod, fmt.Sprintf("%s/", internal)) {
```

Comment:
I wonder if it's better if it's better if we moved lines 201->211 to be above this.